src/cluster.py

```diff
@@ -69,6 +88,14 @@ def _change_owner(self, path: str) -> None:
         # Set the correct ownership for the file or directory.
         os.chown(path, uid=user_database.pw_uid, gid=user_database.pw_gid)
 
+    @property
+    @retry(stop=stop_after_attempt(10), wait=wait_exponential(multiplier=1, min=2, max=10))
+    def cluster_members(self) -> set:
+        """Get the current cluster members."""
+        # Request info from cluster endpoint (which returns all members of the cluster).
+        r = requests.get(f"http://{self.unit_ip}:8008/cluster")
```

Comment:
`requests.get` can raise a lot of different errors, so should these calls to patroni API be wrapped with `try-except`'s or is generelly safe to go without in this context?

Reply:
Yeah, you are right, Paulo. I added some checks to handle that in the next PR commit (https://github.com/canonical/postgresql-operator/pull/11/commits/9939234da34968bc425dc7b2d41e76b47302724a).